enterprise/security/src/test/java/org/neo4j/server/security/enterprise/auth/ConfiguredAuthScenariosInteractionTestBase.java

```diff
@@ -108,27 +109,27 @@ public void shouldWarnWhenUsingNativeAndOtherProvider() throws Throwable
                 .beginTransaction( KernelTransaction.Type.explicit, StandardEnterpriseSecurityContext.AUTH_DISABLED );
         Result result =
                 localGraph.execute( transaction, "EXPLAIN CALL dbms.security.listUsers", Collections.emptyMap() );
-        assertThat(
-                containsNotification( result, "Native user management procedures will not affect non-native users." ),
-                equalTo( true ) );
+        String description = String.format( "%s (%s)", Status.Procedure.ProcedureWarning.code().description(),
+                "dbms.security.listUsers only affect native users." );
```

Comment:
Actually, a better message would be `dbms.security.listUsers only applies to native users.`